details.py

```diff
@@ -0,0 +1,37 @@
+from django.conf import settings
```

Comment:
You can remove some of these imports that aren't being used in this file

Reply:
Ah darn, thanks fer the heads up!